refract-spec.md

```diff
@@ -36,18 +36,21 @@ The Refract Element contains four properties: `element`, `meta`, `attributes`, a
           - `ref` (Link) - Link to referenced element or type
           - `class` (array[string]) - Array of classifications for given element
           - `prefix` (string) - Prefix in which element MAY be found
+          - `name` (enum) - For use when an element is used as a property of an Object Element
+              - string
+              - Element
           - `namespaces` (array[Link]) - Include elements from given namespaces or prefix elements from given namespace
           - `title` (string) - Human-readable title of element
           - `description` (string) - Human-readable description of element
-      - (array[Member Element])
+      - (array[Property Element])
 
 - `attributes` (enum)
 
-    The `attributes` property defines attributes about the given instance of the element, as specified by the `element` property.
+    The `attributes` property defines attributes about the given instance of the element, as specified by the `element` property. When `attributes` is an object, it MAY contain elements itself. The element definition SHOULD be used when interacting with `attributes` and its properites and values.
```

Comment:
> The element definition SHOULD be used when interacting with `attributes` and its properites and values.

@smizell Can you please help me understand this sentence? What does it mean that I should you an element definition when working with attributes?